tests/test_common.py

```diff
@@ -281,10 +282,17 @@ def create_a_fake_process(self, nb_input=1, output_per_input=1, input_name=None,
         :param output_container_udfs: the udfs to be set for the output container (udfs values supports iterable)
         :param step_udfs: the udfs to be set for the StepDetails and the Process (udfs values supports iterable)
         :param next_action: The action to set for the output artifacts -- supports iterable
+        :param process_id: Define the process id
         """
         # Create the projects
         projects = cycle(self.create_fake_projects(**kwargs))
-        p = self.create_instance(Process, uri='p_uri')
+        # Assign process id
+        if kwargs.get('process_id'):
+            uri = kwargs.get('process_id')
+        else:
+            uri = 'p_uri'
+
+        p = self.create_instance(Process, uri=uri)
```

Comment:
Just to be pedantic: 
you could write this with the following code:
```python
p = self.create_instance(Process, uri=kwargs.get('process_id') or 'p_uri')
```
Without the `if`, `else` block.

Reply:
updated